### PR TITLE
Skip semi-colon test with findIf

### PR DESCRIPTION
**Pull request by PabstMirror**
ugly, but the test is failing valid code, so best to disable for now

### CODE REVIEW

**Comment by bux** [comment]
> disable for now

then we might need a reminder to enable this when it has been fixed? 😄

File: tools/sqf_validator.py [hunk 1]
```diff
@@ -117,7 +118,7 @@ def popClosing():
                             if (c not in [' ', '\t', '\n', '/']): # keep reading until no white space or comments
                                 checkForSemiColumn = False
                                 if (c not in [']', ')', '}', ';', ',', '&', '!', '|', '='] and not validKeyWordAfterCode(content, indexOfCharacter)): # , 'f', 'd', 'c', 'e', 'a', 'n', 'i']):
-                                    print("ERROR: Possible missing semi-column ';' detected at {0} Line number: {1}".format(filepath,lineNumber))
+                                    print("ERROR: Possible missing semi-colon ';' detected at {0} Line number: {1}".format(filepath,lineNumber))
```
Comment (dedmen) [inline tools/sqf_validator.py]: https://en.wikipedia.org/wiki/Semicolon
`The semicolon or semi colon` either no `-` or whitespace.

File: tools/sqf_validator.py [hunk 2]
```diff
@@ -57,7 +57,8 @@ def popClosing():
         for c in content:
             if (lastIsCurlyBrace):
                 lastIsCurlyBrace = False
-                checkForSemiColumn = True
+                # Test generates false positives with binary commands that take CODE as 2nd arg (e.g. findIf)
+                checkForSemiColumn = not re.search('findIf', content, re.IGNORECASE)
```
Comment (dedmen) [inline tools/sqf_validator.py]: typo is still in here

**Comment by commy2** [comment]
semi column

**Pull request merged**